at_client/src/main/java/org/atsign/client/util/CameraUtil.java

```diff
@@ -0,0 +1,153 @@
+package org.atsign.client.util;
+
+import com.github.sarxos.webcam.Webcam;
+import org.apache.commons.lang3.StringUtils;
+import org.atsign.common.AtException;
+
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ *
+ * 5/10/2022
+ * Author- Kumar Aggarrwal
+ *
+ * CameraUtil
+ *
+ * This Utility class can be used to capture a stream from the camera and get the result in list of byte[] OR capture a single image from camera into byte[]
+ * Camera name can be provided to chose which camera would be used to capture.
+ *
+ * usage:
+ *
+ *      CameraUtil.getCameraStream(); //For DefaultCamera
+ *      CameraUtil.getCameraStream(String cameraName); for a specific camera
+ *      CameraUtil.getCaptureSingleImage(); //For DefaultCamera
+ *      CameraUtil.getCaptureSingleImage(String cameraName); //For a specific camera
+ *
+ *      //example for camera Name: "Webcam HD Webcam: HD Webcam /dev/video0"
+ *      //Check you system drivers to identify camera name or use getAllCams method here
+ *
+ */
+
+public class CameraUtil{
+
+    public static final String START = "START";
+    public static final String STOP = "STOP";
+    public static final String EXIT = "EXIT";
```

Comment:
I would recommend to use `enum types` instead.

Reply:
This was not being used now, i have deleted them.